patient-view-fe/src/components/header/header.component.tsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import './header.component.css'
+import {PatientBannerComponent} from 'patient-banner'
+import 'patient-banner/dist/index.css'
```

Comment:
Why is it used the CSS file from dist? Do you need it?

Reply:
Yes, I need it.